source/conf.py

```diff
@@ -447,7 +447,19 @@
     # Ignore Twitter links because for some reason valid links sometimes fail
     # with "400 Bad Request"
     r"^https://twitter.com/\w+$",
+    # Ignore support links because we get 403 Client Errors (bad User Agent?)
+    r"^https://support.native-instruments.com/",
+    r"^https://support.serato.com/",
 ]
 
+linkcheck_ignore = [
+    # Unknown why this anchor doesn't appear to work -- could be js-generated.
+    r"^https://github.com/mixxxdj/mixxx/wiki/Midi-Scripting#",
+]
```

Comment:
why are you adding the same option yet again (overwriting the old `linkcheck_ignore` I guess?)

Reply:
copy-paste error.